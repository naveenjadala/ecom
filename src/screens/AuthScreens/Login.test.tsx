import React from 'react';
import {render, screen} from '../../test/test-utils';

import Login from './Login';

describe('Login', () => {
  test('should render Login screen', () => {
    render(<Login />);
    expect(screen).toMatchSnapshot();
  });
});
