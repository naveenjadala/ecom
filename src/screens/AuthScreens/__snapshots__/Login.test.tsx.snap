// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render Login screen 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "minHeight": "100%",
      "minWidth": "100%",
    }
  }
>
  <View>
    <Text>
      Login
    </Text>
  </View>
</RCTSafeAreaView>
`;
